flutter_local_notifications/darwin/Classes/NotificationParser.h

```diff
@@ -8,7 +8,7 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-@interface Converters : NSObject
+@interface NotificationParser : NSObject
```

Comment:
Thanks for the PR. Rather than renaming from Converters to NotificationParser, would you be able to add the `FLN` prefix (an abbreviation of flutter local notifications) so that it's `FLNConverters` instead? Idea came about seeing the official plugins use `FLT` (presumably for Flutter)